components/automate-grpc/protoc-gen-policy/policy/policy.go

```diff
@@ -157,22 +127,14 @@ func (m *Module) Execute(targets map[string]pgs.File, pkgs map[string]pgs.Packag
 }
 
 func (m *Module) processFile(f pgs.File) {
-	// We attempt to extract info and generate code for all policy versions. If
-	// NONE of these attempts are successful, we error out.
-	errors := []error{}
-	for vsn, pol := range policyVersions {
-		out := bytes.Buffer{}
-		err := m.applyTemplate(&out, pol, f)
-		if err != nil {
-			m.Logf("couldn't apply %v template: %s", vsn, err)
-			errors = append(errors, err)
-			continue
-		}
-		generatedFileName := m.ctx.OutputPath(f).SetExt(fmt.Sprintf(".%s-%s.go", moduleName, vsn)).String()
-		m.AddGeneratorFile(generatedFileName, out.String())
-	}
-	if len(errors) == len(policyVersions) {
+	out := bytes.Buffer{}
+	err := m.applyTemplate(&out, policyVersion, f)
+	if err != nil {
+		m.Logf("couldn't apply template: %s", err)
 		m.Fail("code generation failed for all policy versions")
+	} else {
+		generatedFileName := m.ctx.OutputPath(f).SetExt(fmt.Sprintf(".%s.go", moduleName)).String()
```

Comment:
Here we remove the version from the file name generation, which means that generated files went from `*.pb.policy-v2.go` to `*.pb.policy.go`.